src/components/sharing/ShareBox/__tests__/ShareBox-test.js

```diff
@@ -1,23 +1,22 @@
-'use strict';
+jest.disableAutomock();
 
-jest.autoMockOff();
+import React from 'react';
+import ShareBox from '../';
+import ShareIcon from '../../ShareIcon/';
+import TestUtils from 'react-addons-test-utils';
 
 describe('Share Box', function() {
-  var React                  = require('react');
-  var ShareBox               = require('../');
-  var ShareIcon              = require('../../ShareIcon/');
-  var TestUtils              = require('react-addons-test-utils');
-  var scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
+  let scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
 
-  var mockServices = [
+  let mockServices = [
```

Comment:
My query is more targeted at the tool converting var to let. Is it possible to have it convert to const? I wouldn't expect someone to walk through and update 500 instances, just boyscout the files as we go through